lms/djangoapps/grades/signals/handlers.py

```diff
@@ -95,33 +97,25 @@ def recalculate_subsection_grade_handler(sender, **kwargs):  # pylint: disable=u
        - course_id: Unicode string representing the course
        - usage_id: Unicode string indicating the courseware instance
     """
-    try:
-        course_id = kwargs['course_id']
-        usage_id = kwargs['usage_id']
-        student = kwargs['user']
-    except KeyError:
-        log.exception(
-            u"Failed to process SCORE_CHANGED signal, some arguments were missing."
-            "user: %s, course_id: %s, usage_id: %s.",
-            kwargs.get('user', None),
-            kwargs.get('course_id', None),
-            kwargs.get('usage_id', None),
-        )
-        return
-
-    course_key = CourseLocator.from_string(course_id)
+    student = kwargs['user']
```

Comment:
I expect this will have a merge conflict if Cliff's PR merges first, just as a heads up. Are we moving away from using `kwargs.get('x', None)` because we want to bubble the exceptions up anyway? Will these changes make the error messages any less readable?